equinix/data_source_network_device.go

```diff
@@ -275,6 +275,12 @@ func createDataSourceNetworkDeviceSchema() map[string]*schema.Schema {
 			Computed:    true,
 			Description: neDeviceDescriptions["ZoneCode"],
 		},
+		neDeviceSchemaNames["Connectivity"]: {
+			Type:         schema.TypeString,
+			Optional:     true,
+			ValidateFunc: validation.StringInSlice([]string{"PRIVATE"}, true),
```

Comment:
All the enums need to be listed here.

Reply:
This is for data source (GET device by uuid API). Therefore, it's computed. We don't need optional flag here. Computed flag is required.